innerprod/simple/ddh_multi_test.go

```diff
@@ -26,20 +26,30 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestSimple_MultiDDH(t *testing.T) {
+func testSimple_MultiDDHFromParam(t *testing.T, param dDHTestParam) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [don't use underscores in Go names; func testSimple_MultiDDHFromParam should be testSimpleMultiDDHFromParam](https://app.codacy.com/app/tilenmarc/gofe_2/pullRequest?prid=4432579)